openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -663,7 +672,9 @@ public void jckConfigurationForMultijvmWithNoAgent(StfCoreExtension test) throws
 				extraJvmOptions += " -Djdk.xml.maxXMLNameLimit=4000";
 			}	
 			if ( tests.contains("api/org_omg") || tests.contains("api/javax_management") || tests.equals("api") ) {
-				fileContent += "set jck.env.runtime.idl.orbHost " + hostname + ";\n";
+				if (jckVersion.contains("jck8")) {
```

Comment:
If this is limited to jck8 probably moving this if condition out of if condition ` if ( tests.contains("api/org_omg") ...`. So it would be same style as following code?

Or maybe also combine those three if jck8 condition to one?

Reply:
Updated! Thanks